content/docs/started/getting-started-minikf.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "MiniKF"
+description = "A fast and easy way to deploy Kubeflow on your laptop"
+weight = 2
++++
+
+A production-ready, full-fledged, local Kubeflow deployment that
+installs in minutes.
+
+MiniKF is a fast and easy way to get started with Kubeflow. With just a
+few clicks, you are up for experimentation, and for running complete
+Kubeflow Pipelines.
+
+To train at scale, move to a Kubeflow cloud deployment with one click,
+without having to rewrite anything.
+
+See
+[here](https://medium.com/kubeflow/minikf-the-fastest-and-easiest-way-to-deploy-kubeflow-on-your-laptop-a91fb846d0ba)
+for the official announcement and why we started MiniKF.
+
+Join the discussion on the
+[#minikf](https://kubeflow.slack.com/messages/CGRKM3N0G/) Slack channel,
+ask questions, request features, and get support for MiniKF.
+
+### System requirements
+For a smooth experience we recommend that your system meets the
+following requirements:
+  - 12GB RAM
```

Comment:
The bullet formatting is not working - see preview:
https://deploy-preview-534--competent-brattain-de2d6d.netlify.com/docs/started/getting-started-minikf/

Adding an empty line before the first list item should solve this.

Reply:
Added an empty line, thank you!